BLDC.py

```diff
@@ -17,21 +18,24 @@ def __init__(self, pwmPin, directionPin, frequency):
             self.pwmCtrl = PWM.Signal(pwmPin, frequency)
 
     def setDirection(self, direction):
+        self.stop()
+        time.sleep(0.2)
         if direction == 'CW':
             GPIO.write(self.directionPin, 'LOW')
         elif direction == 'CCW':
             GPIO.write(self.directionPin, 'HIGH')
         else:
             raise ValueError("Could not set motor direction, expected either: 'CW' or 'CCW'")
+        time.sleep(0.2)
```

Comment:
do we need to sleep again after the GPIO write? I thought the sleep is just to give time for the motor to fully stop.

Reply:
Whenever there's a GPIO change, it's not instant. It's like a curve. The 0.2 gives it time to reach high/low before doing the next step. If the next step runs in the middle of the curve it might not get the intended results. It's more like "it's better safe than sorry" kind of thing

Reply:
sorry I remember replying to this but don't see the comment. do we need to sleep for every GPIO write? then we should also just move that to `GPIO.write()`

Reply:
It's better to do so cause the GPIO change is not instant. We're doing more testing tomorrow. Testing forward, backward, right, left, ccw, and cw navigation of the 4 motors at once. I'm creating a new class that uses the Motor class. TBH, code is going to change, so for now I'll merge this, but expect another PR with changes tomorrow.